model/src/main/proto/platform_parameter.proto

```diff
@@ -18,6 +18,8 @@ message PlatformParameter {
     // Indicates a string-typed platform parameter.
     string string = 4;
   }
+  // Indicates that the platform parameter is synced
+  bool is_synced = 5;
```

Comment:
For future proofing this, we should probably use an enum instead of a bool. E.g.:

```proto
enum SyncStatus {
  // Indicates that the sync status isn't yet known.
  SYNC_STATUS_UNSPECIFIED = 0;

  // Indicates that the parameter isn't yet synced with the remote server.
  NOT_SYNCED_FROM_SERVER = 1;

  // Indicates the parameter's value has been synced with the remote server.
  SYNCED_FROM_SERVER = 2;
}
```

Then it could be used as so:

```suggestion

  // Indicates the parameter's remote sync status.
  SyncStatus sync_status = 5;
```

This provides a few advantages:
- An enum is more descriptive.
- We get a proper 'default' value (unspecified vs. assuming 'false').
- We can add other sync types in the future without needing to migrate the true/false state.

Reply:
I have switched to this approach. I still can't picture a scenario where the NOT_SYNCED_FROM_SERVER is applicable so it is currently unused but the other two are being used and are working perfectly.

Reply:
I get it now. Thanks.